index.html

```diff
@@ -118,18 +118,25 @@ <h2>Join community calls</h2>
 <div class="w3-padding-64 w3-container w3-light-grey">
   <div class="w3-content">
         <div class="w3-center">
-      <h1>Eclipse OpenJ9 version 0.40.0 released</h1>
-<p>July 2023</p>
+      <h1>Eclipse OpenJ9 version 0.41.0 released</h1>
+<p>October 2023</p>
 </div>
-<p>We're pleased to announce the availability of Eclipse OpenJ9™ v0.40.0.</p>
-<p>This release supports OpenJDK version 8, 11, 17 and 20. For more information about supported platforms and OpenJDK versions, see <a class="w3-hover-opacity" href="https://www.eclipse.org/openj9/docs/openj9_support/" target="_blank">Supported environments</a>.</p>
+<p>We're pleased to announce the availability of Eclipse OpenJ9™ v0.41.0.</p>
+<p>This release supports OpenJDK version 8, 11, and 17. For more information about supported platforms and OpenJDK versions, see <a class="w3-hover-opacity" href="https://www.eclipse.org/openj9/docs/openj9_support/" target="_blank">Supported environments</a>.</p>
 <p>Other updates in this release include the following:</p>
 <ul>
-  <li>Support for persistent shared classes cache is now added on z/OS&reg; systems.</li>
-  <li>A new <code>-XX:codecachetotalMaxRAMPercentage</code> option is added to set the maximum limit for the total JIT code cache size based on the amount of physical memory available to the VM.</li>
-  <li>Even though the container detection fails when you mount the <code>/proc</code> file system with the <code>hidepid=2</code> option on Linux® platforms, the VM starts but with a warning message.</li>
+  <li>The `-XX:+CompactStrings` option is now enabled by default on Java&trade; 11 and later.</li>

```

Comment:
I don't think the backquotes work in html code.

Reply:
I mean, you don't get any formatting change, you just see the backquotes on the web page.